src/ape_init/_cli.py

```diff
@@ -33,6 +33,29 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        git_ignore_path = project_folder / ".gitignore"
+        if git_ignore_path.exists():
```

Comment:
Only matters if it's a file

```suggestion
        if git_ignore_path.is_file():
```

Reply:
Changed because I am looking for a file not a directory